### ISSUE TITLE
window without option to leave

### ISSUE THREAD

**Issue by BrechtCorbeel** [Comfy-Org/ComfyUI-Manager#812]
As per requested to put this issue here, I don't currently care as I am not much using the manager, but in case:

When I am trying stuff to troubleshoot and solve this problem I get: "The custom node DB is currently being updated, and updates to custom nodes are being checked for. NOTE: Update only checks for extensions that have been fetched." at various points trying to do updates and other things to make sure I find a solution, but when "The custom node DB is currently being updated" appears I cannot leave that screen, nothing is happening I can't do anything while it's there, there's no way of clicking it away.

**Comment by ltdrdata** [comment]
What environment are you running ComfyUI in?

**Comment by BrechtCorbeel** [comment]
I'm in Firefox, running Comfy through Smatrix, I guess it is a portable install. My files are on a different (old shitty computer that doesn't work well and is really slow, but still does networking and storage really well) that I use as a NAS.

Everything is updated to latest version except smatrix which can now be upgraded to v2.11.2 from v2.11.1, 

ComfyUI: 2218[bf3e33](2024-05-30)
Manager: V2.35.1

**Comment by ltdrdata** [comment]
What is your OS?
And try without Smatrix.

**Comment by BrechtCorbeel** [comment]
yeah but this is I am pretty sure a thing that is not coded in, it is the update window grey pane that is there and has no way to be removed. I don't remember where but with the manager when you update when it is doing something like updating it does not have an ability to go out of it under a very specific circumstance and if that gets stuck or takes a while you cannot do anything in the GUI anymore except wait.
